test/pytest/test_example_scriptable_tokenzier.py

```diff
@@ -5,19 +5,34 @@
 import sys
 import time
 from argparse import Namespace
+from pathlib import Path
 
 import pytest
 import requests
 import test_utils
 import torch
-from torchtext.models import RobertaClassificationHead
```

Comment:
What is the reason for including the changes to scriptable tokenizers as part of this PR?

Reply:
torchtext upgrade breaks bc so we needed to adjust the unit test

Reply:
same reason as the above